iconloaderlib/src/com/android/launcher3/icons/DotRenderer.java

```diff
@@ -137,14 +139,22 @@ public void draw(Canvas canvas, DrawParams params, int numNotifications) {
         canvas.translate(dotCenterX + offsetX, dotCenterY + offsetY);
         canvas.scale(params.scale, params.scale);
 
+        // Color
+        int dotColor;
+        if (mColor != -1) {
```

Comment:
Should we use another boolean instead to indicate app color? Since white is `-1`.

Reply:
Actually, we could just use `0`.

Reply:
I have set the value of `AppIcon` to `-1`. Isn't `0` black?

Reply:
No. 0 is transparent.

Reply:
Ill change both of the values to 0 then.